src/io/tradetrust/certificate-of-origin/1.0/schema.json

```diff
@@ -0,0 +1,378 @@
+{
+  "$id": "https://schemata.openattestation.com/io/tradetrust/certificate-of-origin/1.0",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Description of data expected for certificate of origin documents",
+  "required": ["iD", "supplyChainConsignment"],
+  "additionalProperties": true,
+  "properties": {
+    "iD": {
+      "type": "string"
+    },
+    "supplyChainConsignment": {
+      "type": "object",
+      "required": ["exporter", "importer"],
+      "properties": {
+        "iD": {
+          "type": "string"
+        },
+        "information": {
+          "type": "string"
+        },
+        "exportCountry": {
+          "type": "object",
+          "properties": {
+            "code": {
+              "type": "string"
+            }
+          }
+        },
+        "exporter": {
```

Comment:
if exporter and importer are the same, can create a definition

Reply:
done.